privates/templates/admin/widgets/clearable_private_file_input.html

```diff
@@ -1,7 +1,11 @@
 {% if widget.is_initial %}
-<p class="file-upload">{{ widget.initial_text }}: <a href="{{ url }}">{{ display_value }}</a>{% if not widget.required %}
+{% if download_allowed %}
+    <p class="file-upload">{{ widget.initial_text }}: <a href="{{ url }}">{{ display_value }}</a>
+{% else %}
+    <p class="file-upload">{{ widget.initial_text }}: {{ display_value }}{% if not widget.required %}
```

Comment:
this `{% if not widget.required %}` appears to not be syntactically correct - this should be caught by a test asserting that the expected output is present in the admin (no download allowed -> `display_value` visible but no anchor tag and vice versa)